<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGenieStaticInterface</class>
 <widget class="QWidget" name="QGenieStaticInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistics Debug</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
	border-style:solid;
	width:90px;
	border-width:1px;
	border-radius: 4px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb( 0, 144, 221), stop:1 rgb( 116, 113, 255));
	color:rgb(255,255,255);
	border-color:rgb( 116, 113, 190);
	
}
QPushButton:hover
{
	border-style:solid;
	width:90px;
	border-width:1px;
	border-radius: 4px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb( 255, 144, 221), stop:1 rgb( 99, 113, 255));
	color:rgb(255,255,255);
	border-color:rgb( 116, 113, 190);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>-114</y>
            <width>358</width>
            <height>228</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QPlainTextEdit" name="ptx_retv"/>
           </item>
           <item>
            <widget class="QLabel" name="lab_retv">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="retstr_lst"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <row>
            <property name="text">
             <string>1</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>简写</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>描述</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>次数</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string>好</string>
            </property>
           </item>
           <item row="0" column="1">
            <property name="text">
             <string>er</string>
            </property>
           </item>
           <item row="0" column="2">
            <property name="text">
             <string>5</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="lab_linfo">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lab_dir">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_4">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QTableWidget" name="tableWidgetRouteType">
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>routemac</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>localmac</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>model</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>firmware</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string>fda</string>
            </property>
           </item>
           <item row="0" column="1">
            <property name="text">
             <string>fdasf</string>
            </property>
           </item>
           <item row="0" column="2">
            <property name="text">
             <string>fdas</string>
            </property>
           </item>
           <item row="0" column="3">
            <property name="text">
             <string>fdasfda</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_3">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="lab_searchstrs">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pagebut">
       <property name="text">
        <string>ChangePage</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="getinfo">
       <property name="text">
        <string>getLocalInfo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="getrouteinfo">
       <property name="text">
        <string>routeinfo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendinstdata">
       <property name="text">
        <string>SendInstData</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendrouterdata">
       <property name="text">
        <string>SendRouterData</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendsearch">
       <property name="text">
        <string>SendSearch</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendclick">
       <property name="text">
        <string>SendClick</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lab_unknown_click">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>retstr_lst</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>lab_retv</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>retstr_lst</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>ptx_retv</receiver>
   <slot>setPlainText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
